server/etcdserver/api/etcdhttp/health.go

```diff
@@ -250,9 +277,11 @@ func (reg *CheckRegistry) runHealthChecks(ctx context.Context, checkNames ...str
 		}
 		if err := check(ctx); err != nil {
 			fmt.Fprintf(&individualCheckOutput, "[-]%s failed: %v\n", checkName, err)
-			h.Health = "false"
```

Comment:
ditto, let's keep it so as not to break the existing `/health` in 3.6.

Reply:
this is not used in /health endpoint. It is only used for livez/healthz.